### DISCUSSION TITLE
top ten sussy fortnite balls

### DISCUSSION THREAD

**Discussion by nekoify** [aeolus-1/triangleGame.github.io#23] [General]
1. Michael Jackson
2. Obama
3. Ninja from fortnite
4. white stuff
5.  Amog us character
6. thanks for watching

**Comment by aeolus-1** [comment]
corey hankison is pretty sussy as well

**Reply by Quantaviousdingleberry420** [reply]
thats kinda gay

**Comment by Quantaviousdingleberry420** [comment]
Michael Jackson has such big balls 😳 🥱

**Reply by nekoify** [reply]
true

**Comment by nekoify** [comment]
my goofy ahh uncle is holding me captive
